tests/integration/tests/autocomplete.js

```diff
@@ -250,7 +250,7 @@ test('Search Query', async () => {
   const searchValue = await getInputValue(page, '#search');
 
   // search input is filled with query
-  expect(searchValue).toEqual(searchQuery);
+  expect(searchValue).toEqual("Musée d'Orsay");
```

Comment:
If this is the expected behavior, then the comment message is obsolete and should be update too.

But is it really an expected change in this case? I fear that it could cause frustration to the user that can't complete the query anymore, after the input value has been overridden with a possibly irrelevant result name.

Reply:
Sorry this is just a residue of when I fiddled with the tests trying to make them work again. I think I can keep it as it was before